Gems/PhysX/Code/Tests/CharacterControllerTests.cpp

```diff
@@ -159,6 +159,115 @@ namespace PhysX
         }
     }
 
+    TEST_F(PhysXDefaultWorldTest, CharacterController_GamePlayComponenet_GravityMovesEntity)
```

Comment:
suggestion/nit: Test name could instead be.. `EntityFallsUnderGravity`

Reply:
Updated